### PR TITLE
pt-br: Correcting Brasilian Portuguese errors and small improvements.

### PR DESCRIPTION
**Pull request by ghost**
### Corrections:

1. Fixed Brasilian Portuguese errors.
2. Small improvements.

### CODE REVIEW

File: files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md [hunk 1]
```diff
@@ -201,22 +202,23 @@ Há vários novos e excitantes formatos de imagens (como WebP e JPEG-2000) que p
...
+- **Mudança de resolução:** O problema é apresentar arquivos menores de imagens para dispositivos estreitos, porque eles não precisam de imagens gigantes como em computadores - e também, opcionalmente, que você queira apresentar imagens de diferentes resoluções para altas e baixas densidades de tela. Isto pode resolver usando [vector graphics](/pt-BR/docs/Learn/HTML/Multimedia_and_embedding/Adding_vector_graphics_to_the_Web) (imagens SVG), e os atributos [`srcset`](/pt-BR/docs/Web/HTML/Element/img#srcset) e [`sizes`](/pt-BR/docs/Web/HTML/Element/img#sizes).
 
-Isto também encerra o módulo [Multimedia and embedding](/pt-BR/docs/Learn/HTML/Multimedia_and_embedding)! A única coisa para fazer agora é seguir e tentar nosso teste de multimídia, e ver como você está. Divirta-se!
+Isto também encerra o módulo [Multimídia e Incorporação](/pt-BR/docs/Learn/HTML/Multimedia_and_embedding)! A única coisa para fazer agora é seguir e tentar nosso [Teste de Multimídia e Incorporação](/pt-BR/docs/Learn/HTML/Multimedia_and_embedding/Mozilla_splash_page), e ver como você está. Divirta-se!
```
Comment (raphash) [inline files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md:221]: I translated the _Multimedia and embedding_ and added the relational link to the future translation of _Mozilla splash page_!

File: files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md [hunk 2]
```diff
@@ -201,22 +202,23 @@ Há vários novos e excitantes formatos de imagens (como WebP e JPEG-2000) que p
...
-2. Encontre uma boa imagem ampla de um panorama com algum detalhe contido em alguma parte. Crie uma versão de tamanho web usando um editor de imagem, então coloque para mostrar uma pequena parte que amplia o detalhe, e salve em uma segunda imagem (algo como 480px está bom).
+1. Escreva um simples HTML contendo seu código (use `not-responsive.html` como um ponto de partida, se quiser).
+2. Encontre uma boa imagem ampla de um panorama com algum detalhe contido em alguma parte. Crie uma versão de tamanho web usando um editor de imagem, então coloque para mostrar uma pequena parte que amplia o detalhe, e salve como uma segunda imagem (algo como 480px está bom).
 3. Use o elemento `<picture>` para implementar uma mudança de imagem!
 4. Crie múltiplos arquivos de imagem de diferentes tamanhos, cada um mostrando a mesma imagem.
 5. Use `srcset`/`size` para criar um exemplo de mudança de resolução, que sirva para os mesmos tamanhos de imagens em diferentes resoluções, ou diferentes tamanhos de imagens em cada largura de janela.
 
-> **Nota:** Use a ferramenta de desenvolvimento do navegador para ajudar a ver os tamanhos que você precisa, como mencionado acima.
```
Comment (raphash) [inline files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md:210]: I solved the extra note problem and added the [!NOTE] block correctly.

File: files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md [hunk 3]
```diff
@@ -201,22 +202,23 @@ Há vários novos e excitantes formatos de imagens (como WebP e JPEG-2000) que p
...
 
-1. Escreva um simples HTML contendo seu código (use `not-responsive.html` como um ponto de partida, se quiser)
-2. Encontre uma boa imagem ampla de um panorama com algum detalhe contido em alguma parte. Crie uma versão de tamanho web usando um editor de imagem, então coloque para mostrar uma pequena parte que amplia o detalhe, e salve em uma segunda imagem (algo como 480px está bom).
+1. Escreva um simples HTML contendo seu código (use `not-responsive.html` como um ponto de partida, se quiser).
+2. Encontre uma boa imagem ampla de um panorama com algum detalhe contido em alguma parte. Crie uma versão de tamanho web usando um editor de imagem, então coloque para mostrar uma pequena parte que amplia o detalhe, e salve como uma segunda imagem (algo como 480px está bom).
 3. Use o elemento `<picture>` para implementar uma mudança de imagem!
 4. Crie múltiplos arquivos de imagem de diferentes tamanhos, cada um mostrando a mesma imagem.
 5. Use `srcset`/`size` para criar um exemplo de mudança de resolução, que sirva para os mesmos tamanhos de imagens em diferentes resoluções, ou diferentes tamanhos de imagens em cada largura de janela.
```
Comment (raphash) [inline files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md:210]: I solved the extra note problem and added the [!NOTE] block correctly.

File: files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md [hunk 4]
```diff
@@ -171,15 +172,15 @@ Este código nos permite mostrar uma imagem adequada em ambas extensas e estreit
 ![Our example site as viewed on a wide screen - here the first image works ok, as it is big enough to see the detail in the center.](picture-element-wide.png)![Our example site as viewed on a narrow screen with the picture element used to switch the first image to a portrait close up of the detail, making it a lot more useful on a narrow screen](picture-element-narrow.png)
```
Comment (raphash) [inline files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md:173]: I fixed a problem that had an extra **note** inside the note block.

File: files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md [hunk 5]
```diff
@@ -97,33 +97,34 @@ Então, com estes atributos no lugar, o navegador irá:
...
 
 Eu configurei a largura da janela para 320px e depois 480px; para cada uma eu fui no [DOM Inspector](/pt-BR/docs/Tools/Page_Inspector), cliquei no elemento {{htmlelement("img")}} no qual nós estamos interessados, então olhei o tamanho na aba Box Model view no lado direito da tela. Isto deve dar para você a dica da largura de imagem que você precisa.
 
 ![A screenshot of the firefox devtools with an image element highlighted in the dom, showing its dimensions as 440 by 293 pixels.](box-model-devtools.png)
 
 Próximo, você pode checar se o `srcset` está funcionando configurando a largura da janela para a qual você quiser (coloque para uma largura estreita, por exemplo), abrindo o Network Inspector (Ferramentas > Web Developer > Network), então recarregue a página. Isto deve dar a você uma lista do que foi carregado na página, e aqui você pode checar qual arquivo da imagem foi escolhida para baixar.
```
Comment (raphash) [inline files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md:119]: fix [!NOTE].

File: files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md [hunk 6]
```diff
@@ -201,22 +202,23 @@ Há vários novos e excitantes formatos de imagens (como WebP e JPEG-2000) que p
...
+- **Mudança de resolução:** O problema é apresentar arquivos menores de imagens para dispositivos estreitos, porque eles não precisam de imagens gigantes como em computadores - e também, opcionalmente, que você queira apresentar imagens de diferentes resoluções para alta e baixa densidades de tela. Isto pode ser resolvido usando [vector graphics](/pt-BR/docs/Learn/HTML/Multimedia_and_embedding/Adding_vector_graphics_to_the_Web) (imagens SVG), e os atributos [`srcset`](/pt-BR/docs/Web/HTML/Element/img#srcset) e [`sizes`](/pt-BR/docs/Web/HTML/Element/img#sizes).
 
-Isto também encerra o módulo [Multimedia and embedding](/pt-BR/docs/Learn/HTML/Multimedia_and_embedding)! A única coisa para fazer agora é seguir e tentar nosso teste de multimídia, e ver como você está. Divirta-se!
+Isto também encerra o módulo [Multimídia e Incorporação](/pt-BR/docs/Learn/HTML/Multimedia_and_embedding)! A única coisa para fazer agora é seguir e tentar nosso [Teste de Multimídia e Incorporação <sup>(inglês)</sup>](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Mozilla_splash_page), e ver como você está. Divirta-se!
```
[the remark this answers is not in the document]
Comment (raphash) [reply files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md]: **I already fixed that** i removed the absolute link and put a relational link in pt-br for a future translation.

File: files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md [hunk 7]
```diff
@@ -5,54 +5,53 @@ slug: Learn/HTML/Multimedia_and_embedding/Responsive_images
...
 
-> [!NOTE]
-> As novas funcionalidades discutidas neste artigo — [`srcset`](/pt-BR/docs/Web/HTML/Element/img#srcset)/[`sizes`](/pt-BR/docs/Web/HTML/Element/img#sizes)/{{htmlelement("picture")}} — são todas suportadas nas versões atuais de navegadores mobile e desktop (incluindo Microsoft Edge, embora não suportada pelo Internet Explorer).
+> **Nota:** As novas funcionalidades discutidas neste artigo — [`srcset`](/pt-BR/docs/Web/HTML/Element/img#srcset)/[`sizes`](/pt-BR/docs/Web/HTML/Element/img#sizes)/{{htmlelement("picture")}} — são todas suportadas nas versões atuais de navegadores mobile e desktop (incluindo Microsoft Edge, embora não suportada pelo Internet Explorer).
```
Comment (josielrocha) [inline files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md:40]:
```suggestion
> [!NOTE]
> As novas funcionalidades discutidas neste artigo — [`srcset`](/pt-BR/docs/Web/HTML/Element/img#srcset)/[`sizes`](/pt-BR/docs/Web/HTML/Element/img#sizes)/{{htmlelement("picture")}} — são todas suportadas nas versões atuais de navegadores mobile e desktop (incluindo Microsoft Edge, embora não suportada pelo Internet Explorer).
```

File: files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md [hunk 8]
```diff
@@ -90,40 +89,38 @@ Os atributos `srcset` e `sizes` parecem complicados, mas não são difíceis de
 2. Um espaço.
 3. A **largura do slot** que a imagem irá preencher quando a condição de mídia for verdadeira (`440px`).
 
-> [!NOTE]
-> Para a largura do slot, você pode fornecer um tamanho absoluto (`px`, `em`) ou um tamanho relativo (como porcentagem). Você pode ter notado que o último slot de largura não tem condição de mídia - isto é o padrão que será escolhido quando nenhuma condição for verdadeira. O navegador ignora tudo depois da primeira condição satisfeita, então tenha cuidado com a ordem de condições.
+> **Nota:** Para a largura do slot, você pode fornecer um tamanho absoluto (`px`, `em`) ou um tamanho relativo (como porcentagem). Você pode ter notado que o último slot de largura não tem condição de mídia - isto é o padrão que será escolhido quando nenhuma condição for verdadeira. O navegador ignora tudo depois da primeira condição satisfeita, então tenha cuidado com a ordem de condições.
```
Comment (josielrocha) [inline files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md:92]:
```suggestion
> [!NOTE] 
> Para a largura do slot, você pode fornecer um tamanho absoluto (`px`, `em`) ou um tamanho relativo (como porcentagem). Você pode ter notado que o último slot de largura não tem condição de mídia - isto é o padrão que será escolhido quando nenhuma condição for verdadeira. O navegador ignora tudo depois da primeira condição satisfeita, então tenha cuidado com a ordem de condições.
```

File: files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md [hunk 9]
```diff
@@ -90,40 +89,38 @@ Os atributos `srcset` e `sizes` parecem complicados, mas não são difíceis de
...
 
 Navegadores antigos que não suportam estas funcionalidades serão ignorados, seguiremos e carregaremos a imagem definida no atributo [`src`](/pt-BR/docs/Web/HTML/Element/img#src) como normal.
 
-> [!NOTE]
-> No {{htmlelement("head")}} do documento você encontrará a linha `<meta name="viewport" content="width=device-width">`: isto força navegadores de celular adotar a largura real para carregar páginas web (alguns navegadores mobile mentem sobre sua largura da janela, e em vez carregam páginas em uma largura grande e então encolhem a página carregada, o que é de muita ajuda para imagens e design responsivos. Nós iremos ensinar mais sobre isso em um módulo futuro).
+> **Nota:** No {{htmlelement("head")}} do documento você encontrará a linha `<meta name="viewport" content="width=device-width">`: isto força os navegadores de celular a adotar a largura real para carregar páginas da web (alguns navegadores mobile mentem sobre sua largura da janela, e em vez carregam páginas em uma largura grande, então encolhem a página carregada, o que é de muita ajuda para imagens e designs responsivos. Nós iremos ensinar mais sobre isso em um módulo futuro).
```
Comment (josielrocha) [inline files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md:105]:
```suggestion
> [!NOTE]
> No {{htmlelement("head")}} do documento você encontrará a linha `<meta name="viewport" content="width=device-width">`: isto força os navegadores de celular a adotar a largura real para carregar páginas da web (alguns navegadores mobile mentem sobre sua largura da janela, e em vez carregam páginas em uma largura grande, então encolhem a página carregada, o que é de muita ajuda para imagens e designs responsivos. Nós iremos ensinar mais sobre isso em um módulo futuro).
```

File: files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md [hunk 10]
```diff
@@ -90,40 +89,38 @@ Os atributos `srcset` e `sizes` parecem complicados, mas não são difíceis de
...
 
 ![A screenshot of the firefox devtools with an image element highlighted in the dom, showing its dimensions as 440 by 293 pixels.](box-model-devtools.png)
 
 Próximo, você pode checar se o `srcset` está funcionando configurando a largura da janela para a qual você quiser (coloque para uma largura estreita, por exemplo), abrindo o Network Inspector (Ferramentas > Web Developer > Network), então recarregue a página. Isto deve dar a você uma lista do que foi carregado na página, e aqui você pode checar qual arquivo da imagem foi escolhida para baixar.
 
-> **Nota:**Use o Mozilla Firefox para testar `srcset`. O Chrome carrega a melhor imagem se estiver em cache no navegador, anulando o propósito do teste na ferramenta de desenvolvimento.
+> **Nota:** Use o Mozilla Firefox para testar o `srcset`. O Chrome carrega a melhor imagem se estiver em cache no navegador, anulando o propósito do teste na ferramenta de desenvolvimento.
```
Comment (josielrocha) [inline files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md:117]:
```suggestion
> [!NOTE]
> Use o Mozilla Firefox para testar o `srcset`. O Chrome carrega a melhor imagem se estiver em cache no navegador, anulando o propósito do teste na ferramenta de desenvolvimento.
```

File: files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md [hunk 11]
```diff
@@ -162,24 +159,23 @@ Vamos consertar isso, com {{htmlelement("picture")}}! Como [`<video>` e `<audio>
...
 - Em todos os casos, você deve fornecer um elemento `<img>`, com `src` e `alt`, logo antes do `</picture>`, de outra forma não aparecerá imagens. Assim um padrão será aplicado quando nenhuma condição for atendida (você pode remover o segundo elemento `<source>` neste exemplo), e verificar navegadores que não suportam o elemento `<picture>`.
 
 Este código nos permite mostrar uma imagem adequada em ambas extensas e estreitas telas, como podemos ver abaixo:
 
 ![Our example site as viewed on a wide screen - here the first image works ok, as it is big enough to see the detail in the center.](picture-element-wide.png)![Our example site as viewed on a narrow screen with the picture element used to switch the first image to a portrait close up of the detail, making it a lot more useful on a narrow screen](picture-element-narrow.png)
 
-> [!NOTE]
-> Nota: Você deveria usar o atributo `media` somente em cenários de direção de arte; quando você usa `media`, não oferecendo também condições com o atributo `sizes`.
+> **Nota:** Você deveria usar o atributo `media` somente em cenários de direção de arte; quando você usa `media`, não oferecendo também condições com o atributo `sizes`.
```
Comment (josielrocha) [inline files/pt-br/learn/html/multimedia_and_embedding/responsive_images/index.md:170]:
```suggestion
> [!NOTE]
> Você deveria usar o atributo `media` somente em cenários de direção de arte; quando você usa `media`, não oferecendo também condições com o atributo `sizes`.
```

**Pull request merged**